lib/features/family/features/reflect/presentation/pages/grateful_screen.dart

```diff
@@ -170,8 +174,14 @@ class _GratefulScreenState extends State<GratefulScreen> {
 
   Future<void> _navigateToChildGivingScreen(
     BuildContext context,
-    GameProfile profile,
+    GameProfile profile, Organisation organisation,
   ) async {
+    final generatedMediumId =
+    base64.encode(organisation.namespace.codeUnits);
+    await context
+        .read<CollectGroupDetailsCubit>()
+        .getOrganisationDetails(generatedMediumId);
+    context.read<ScanNfcCubit>().stopScanningSession();
```

Comment:
is this really necessary?

Reply:
I don't think it is in hindsight, will remove it.